analysis/import/GitLogParser/src/test/kotlin/de/maibornwolff/codecharta/importer/gitlogparser/GitLogParserTest.kt

```diff
@@ -62,4 +82,18 @@ class GitLogParserTest {
 
         assertTrue(file.exists())
     }
+
+    @ParameterizedTest
+    @MethodSource("provideValidInputFiles")
+    fun `should return true with valid git repository as input for isUsable`(inputFile: String) {
```

Comment:
I would not call the method name `isUsable` in the test name because this causes a depencency/connascence (https://www.maibornwolff.de/know-how/connascence-regeln-fuer-gutes-software-design/)

Lets name the test from a business perspective and omit the technical "HOW":
```suggestion
    fun `should be identified as applicable for given directory path containing a git folder`(inputFile: String) {
```

more candidates would be `should NOT be identified as applicable if no git folder is present at given path`